.github/workflows/verify.yml

```diff
@@ -57,11 +59,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
       - uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # v5.3.0
         with:
-          go-version: '1.23'
+          go-version-file: './go.mod'
```

Comment:
https://github.com/sigstore/sigstore/pull/1958 added a strategy in case we want each go version to differ. Thoughts?

Reply:
this is really only used AFAIK to help install golangci-lint